### PR TITLE
Trap errors on beta hooks

### PR DESCRIPTION
**Pull request by marc0der**
This will prevent further execution of install/selfupdate if any error occurs.

### CODE REVIEW

**Comment by helpermethod** [comment]
Left a few comments, nothing fancy.

**Comment by marc0der** [comment]
@helpermethod I'm not seeing the comments. Did you submit the review?

**Review by helpermethod — CHANGES_REQUESTED** [review]

File: app/views/includes/trap_errors.scala.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+set -e
+
+#keep track of the last executed command
+trap 'last_command=$current_command; current_command=$BASH_COMMAND' DEBUG
```
[the remark this answers is not in the document]
Comment (marc0der) [reply app/views/includes/trap_errors.scala.txt:4]: AFAIK, these scripts are both run with bash so it should work.

**Comment by helpermethod** [comment]
Comments should be visible now.

**Review by helpermethod — APPROVED** [review]

**Pull request merged**